base_agent/memory_nodes.py

```diff
@@ -763,22 +762,41 @@ class TaskNode(MemoryNode):
         >>> TaskNode(agent_memory=agent_memory, memid=memid)
     """
 
-    TABLE_COLUMNS = ["uuid", "action_name", "pickled", "paused", "created_at", "finished_at"]
+    TABLE_COLUMNS = [
+        "uuid",
+        "action_name",
+        "pickled",
+        "prio",
+        "running",
+        "paused",
+        "created_at",
+        "finished_at",
+    ]
     TABLE = "Tasks"
     NODE_TYPE = "Task"
 
     def __init__(self, agent_memory, memid: str):
         super().__init__(agent_memory, memid)
-        pickled, created_at, finished_at, action_name = self.agent_memory._db_read_one(
-            "SELECT pickled, created_at, finished_at, action_name FROM Tasks WHERE uuid=?", memid
-        )
+        try:
+            pickled, prio, running, created_at, finished_at, action_name = self.agent_memory._db_read_one(
+                "SELECT pickled, prio, running, created_at, finished_at, action_name FROM Tasks WHERE uuid=?",
+                memid,
+            )
+        except:
```

Comment:
we need better error handling here and shouldn't check in our ipdb set_trace :)

Reply:
lol